oktapam/constants/descriptions/attributes.go

```diff
@@ -65,24 +65,24 @@ var (
-	ServerAccess                     = "If 'true', members of this ASA Group have access to the ASA Project servers."
-	ServerAdmin                      = "If 'true', members of ASA Group have sudo permissions on ASA Project servers."
+	RDPSessionRecording              = "If `true`, enable remote desktop protocol (RDP) recording on all servers in the ASA Project."
+	RefuseConnections                = "If `true`, gateway refuse connection."
+	RemovedAt                        = "UTC time of resource removal from parent resource. Format is '2022-01-01 00:00:00 +0000 UTC'."
+	RequirePreauthForCreds           = "If `true`, require preauthorization before an ASA User can retrieve credentials to sign in."
+	Roles                            = "A list of roles for the ASA Group. Options are 'access_user', 'access_admin', and 'reporting_user'."
+	ServerAccess                     = "If `true`, members of this ASA Group have access to the ASA Project servers."
+	ServerAdmin                      = "If `true`, members of ASA Group have sudo permissions on ASA Project servers."
 	ServerUserName                   = "The name of the corresponding ASA Server User."
 	ServersSelector                  = "Enables access to ASA Servers with labels matching all selectors. Only available to customers that have the Early Access Policy Sync feature enabled on their team."
 	ServiceAccountUsername           = "The username of the service account that can be used to query the domain."
 	ServiceAccountPassword           = "The password of the service account that can be used to query the domain."
-	SSHCertificateType               = fmt.Sprintf("The SSH certificate type used by access requests. Options include: [%s]. `%s` is a deprecated key algorithm type. "+
-		"This option should only be used to connect to legacy systems that cannot use newer SSH versions. If you do need to use `%s`, it is recommended to connect via a gateway with traffic forwarding. "+
-		"Otherwise, please use a more current key algorithm. If left unspecified, `%s` is used by default.", typed_strings.SSHCertTypeListFormat(), typed_strings.CertTypeRsa, typed_strings.CertTypeRsa, typed_strings.CertTypeEd25519)
-	SSHSessionRecording = "If 'true', enables ssh recording on server access requests."
-	Status              = "The status of the ASA User. Valid statuses are `ACTIVE`, `DISABLED`, and `DELETED`."
+	SSHCertificateType               = fmt.Sprintf("The SSH certificate type used by access requests. Options include: [%s]. '%s' is a deprecated key algorithm type. "+
+		"This option should only be used to connect to legacy systems that cannot use newer SSH versions. If you do need to use '%s', it is recommended to connect via a gateway with traffic forwarding. "+
+		"Otherwise, please use a more current key algorithm. If left unspecified, '%s' is used by default.", typed_strings.SSHCertTypeListFormat(), typed_strings.CertTypeRsa, typed_strings.CertTypeRsa, typed_strings.CertTypeEd25519)
```

Comment:
If we're changing the cert types to be wrapped in single question, this `ListFormat()` method will need to be changed to reflect that as well.